meson.build

```diff
@@ -2417,9 +2417,13 @@ ukify = custom_target(
         command : [jinja2_cmdline, '@INPUT@', '@OUTPUT@'],
         install : want_ukify,
         install_mode : 'rwxr-xr-x',
-        install_dir : libexecdir)
+        install_dir : bindir)
```

Comment:
Update synopsis to reflect this too?

Reply:
Oh, right. I also updated the path in `60-ukify.install`. I think that covers all the spots, because `git grep /usr/lib/systemd/ukify` returns no matches.